salt/modules/lxd.py

```diff
@@ -1376,14 +1376,7 @@ def container_device_get(
 
 
 def container_device_add(
-    name,
-    device_name,
-    device_type="disk",
-    remote_addr=None,
-    cert=None,
-    key=None,
-    verify_cert=True,
-    **kwargs
+    name, device_name, remote_addr=None, cert=None, key=None, verify_cert=True, **kwargs
```

Comment:
looks like `device_type` is being removed. We cannot remove this kwarg without properly deprecating it. Can we detect the version of hte library and allow this kwarg to still be used if on the older version?